tools/lambdaminer/get_projects.xml

```diff
@@ -0,0 +1,45 @@
+<tool id="lm_get_projects" name="Get LMDB Projects" version="0.1.0+galaxy0" profile="21.05">
+    <requirements>
+        <requirement type="package" version="2.2.2">pandas</requirement>
+        <requirement type="package" version="2.9.9">psycopg2</requirement>
+        <requirement type="package" version="2.0.37">sqlalchemy</requirement>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+        #if $test != ""
+            export LAMBDAMINER_CREDENTIALS="$__tool_directory__/test-data/test_credentials.json";
+        #end if
+        python '$__tool_directory__/get_projects.py'
+        --login-name '$__user_name__'
+        --output-file '$projects'
+    ]]></command>
+    <inputs>
+        <param name="test" type="hidden"/>
+        <param name="login" type="text"/>
```

Comment:
I think if you remove this parameter and the `label` attribute of the outout we should be done here. 

If you like you could add a creator tag: 
https://docs.galaxyproject.org/en/master/dev/schema.html#tool-creator-organization